code/controllers/configuration.dm

```diff
@@ -517,6 +518,8 @@
 					config.force_random_names		= 1
 				if("allow_ai")
 					config.allow_ai					= 1
+				if("disable_secborg")
+					config.allow_secborg			= 0

```

Comment:
Would be better if config name was consistent, instead of changing from disable_secborg to allow_secborg